lighthouse-cli/test/smokehouse/test-definitions/oopif-scripts.js

```diff
@@ -105,14 +105,33 @@ const expectations = {
       {
         src: 'http://localhost:10200/simple-script.js',
         source: 'network',
-        content: /🪁/,
       },
       {
         src: 'http://localhost:10503/simple-script.js',
         source: 'network',
-        content: /🪁/,
       },
     ],
+    Scripts: {
+      _includes: [
+        {
+          url: 'http://localhost:10200/simple-script.js',
+          content: /🪁/,
+        },
+        {
+          url: 'http://localhost:10200/oopif-simple-page.html',
```

Comment:
These inline scripts weren't caught by ScriptElements because it doesn't look inside iframe elements. But! they are still in the same process (being from the same host), so they don't get excluded.